pre-processing/primod/ribamod/exchange_creator.py

```diff
@@ -0,0 +1,172 @@
+import numpy as np
+import pandas as pd
+import xarray as xr
+from numpy.typing import NDArray
+from scipy.spatial import KDTree
+
+Int = np.int_
+Float = np.float_
+Bool = np.bool_
+
+
+def check_conductance(conductance: xr.DataArray) -> xr.DataArray:
```

Comment:
This doesn't seem to check anything at the moment :)
Maybe give it a different name for now?